### ISSUE TITLE
is there a way to continue the installation?

### ISSUE THREAD

**Issue by bonkxd1234** [n00mkrad/flowframes#65]
the download stucks suddenly for no reason im having a good internet

**Comment by HackerDaGreat57** [comment]
Yes, my installation (currently happening) is downloading at 200kbps while my internet connection is capable of 50mbps
